src/libraries/tests.proj

```diff
@@ -188,6 +194,9 @@
 
     <!-- https://github.com/dotnet/runtime/issues/50493 -->
     <ProjectExclusions Include="$(RepoRoot)\src\tests\FunctionalTests\Android\Device_Emulator\AOT\Android.Device_Emulator.Aot.Test.csproj" />
+    
+    <!-- Out of memory https://github.com/dotnet/runtime/issues/66831 -->
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Security.Cryptography.Tests\tests\System.Security.Cryptography.Tests.csproj" />
```

Comment:
instead of duplicating hte line, why not make a new block `<ItemGroup Condition="'$(TargetOS)' == 'Android' and '$(RunDisabledAndroidTests)' != 'true'">` with it in once.

Reply:
oh I see Mitchell's comment.

Reply:
We can use the block on L109. I agree, it would be safer to disable for all of Android given that this memory issue seems flakey.